roles/girder/tasks/main.yml

```diff
@@ -90,13 +90,10 @@
     sudo: yes
     sudo_user: "{{ grits_user }}"
 
-  - name: Install grunt globally
-    npm: name={{ item }} global=yes
-    with_items:
-      - grunt
-      - grunt-cli
-    sudo: yes
-    sudo_user: root
+  - name: "Ensure grunt is installed"
+    command: npm -g install grunt grunt-cli || true
```

Comment:
That was something I forgot to remove. Fixed that with this commit:
https://github.com/ecohealthalliance/grits-deploy-ansible/commit/6c752eb89a49be3ff6d03627119c0249432831ed